Makefile

```diff
@@ -30,6 +37,10 @@ build:
 test:
 	go test -race -mod=readonly ./...
 
+.PHONY: generate
+generate: bin/mockgen
```

Comment:
should I add this to default actions we run? if so where

Reply:
I wouldn't do it as we are committing mocks. But doesn't matter too much, you can add it if you want